File: test/twoLines.R
require(ggplot);

let x = seq(-5,5, by = 0.2);
let y = sin(x);
let z = cos(y);

print(x);

bitmap(file = `${@dir}/line_3colors.png`, width = 2600, height = 1600) {
    ggplot(data.frame(x = x, sin =y, cos = z), padding = "padding: 200px 600px 200px 200px;") 
    + geom_line( aes(x = "x", y = "sin"), width = 8)
	 + geom_line( aes(x = "x", y = "cos"), width = 8)
	  + geom_line( aes(x = "x", y = "~sin + cos"), width = 8)
    ;
}

File: test/twolines.jl
using ggplot

x = seq(-5,5, by = 0.2)
y = sin(x)
z = cos(y)

table = data.frame(x = x, sin =y, cos = z)

print("preview of the input data:")
print(table, max.print = 13)

bitmap(plt, file = `${@dir}/line_3colors_julia.png`):

	plt = ggplot(table, padding = "padding: 225px 600px 300px 250px;") 
	plt += geom_line( aes(x = "x", y = "sin"), width = 8)
	plt += geom_line( aes(x = "x", y = "cos"), width = 8)
	plt += geom_line( aes(x = "x", y = "~sin + cos"), width = 8)
    plt += ggtitle("line overlaps")
    plt += xlab("time(ticks)") 
    plt += ylab("signal data")

    plot(plt, width = 2600, height = 1600)

end